services/api-server/tests/unit/_with_db/test_product.py

```diff
@@ -29,22 +29,22 @@ async def test_product_webserver(
     mocked_webserver_rest_api_base: respx.MockRouter,
     create_fake_api_keys: Callable[[PositiveInt], AsyncGenerator[ApiKeyInDB, None]],
     faker: Faker,
-) -> None:
+):
     assert client
 
-    keys: dict[int, ApiKeyInDB] = {}
-    wallet_id: int = faker.pyint(min_value=1)
-    async for key in create_fake_api_keys(2):
-        wallet_id += faker.pyint(min_value=1)
-        keys[wallet_id] = key
+    wallet_to_api_keys_map: dict[int, ApiKeyInDB] = {}
+    wid: int = faker.pyint(min_value=1)
```

Comment:
wid... are we back into the 80s?
no need to cast to int here.

Reply:
:-) I wanted to distingusih from the other wallet_id.